### PR TITLE
Mut table

### PR DESCRIPTION
**Pull request by ArtPoon**

### CODE REVIEW

**Review by ArtPoon — COMMENTED** [review]
why did `.gitmodules` get deleted?

**Comment by ArtPoon** [comment]
We should check usage rights on the icons being used here @bonnielu

**Comment by GopiGugan** [comment]
> why did `.gitmodules` get deleted?

Looks like the the links to the submodules in `covizu/covizu/data` are also missing. I'll merge these into this branch from `dev` and then should we do a dry run?

**Comment by ArtPoon** [comment]
Go ahead thanks.  There's a small change to text in HTML I need to make as well

File: batch.py [hunk 1]
```diff
@@ -162,9 +162,10 @@ def process_feed(args, callback=None):
     # get mutation info
     locator = SC2Locator()
     mutations = {}
+    print(get_mutations(by_lineage).items())
```
Comment (ArtPoon) [inline batch.py:165]: print statements should be removed before code can be merged to production branch

**Comment by bonnielu** [comment]
> We should check usage rights on the icons being used here @bonnielu

All of the shape icons are public domain access. Just checked the bat/antibody icons and those require attribution. The images are small enough that embedded attribution isn't required. Would adding a reference in the code suffice? Or should I just try to replace those with icons that are of public domain.

**Comment by ArtPoon** [comment]
I'd rather replace those with public domain alternatives.  What about this?
https://commons.wikimedia.org/wiki/File:Bat-shadow.svg

**Comment by ArtPoon** [comment]
I just generated a modified version of this public domain SVG:
https://commons.wikimedia.org/wiki/File:IgY_antibody.svg
IgY_antibody_bw

**Comment by GopiGugan** [comment]
* temporarily disabled cron job
* checked out `mut_table` branch
* `python3 setup.py install --user`
* `module load openmpi/gnu`
* `nohup python3 batch.py --dry-run > pr-381.log &`

**Comment by GopiGugan** [comment]
Run was successful:

```console
[covizu@BEVi covizu]$ tail -10 pr-381.log 
🏄 [20:46:37.503937] start C.37, 6470 entries
🏄 [20:58:09.112812] start AY.99.2, 16387 entries
🏄 [21:08:38.467711] start AY.69, 10782 entries
🏄 [21:17:52.558898] start B.1.221, 10417 entries
🏄 [21:28:36.125090] start B.1.1.214, 17591 entries
🏄 [21:37:58.303211] start B.1.1.284, 8973 entries
🏄 [21:41:41.719814] start B.1.438.1, 6652 entries
🏄 [21:44:42.563823] start D.2, 9251 entries
🏄 [21:46:20.003546] Parsing output files
🏄 [1 day, 0:05:45.013480] All done!
```

**Pull request merged**